[css-typed-om] #350 Rename styleMap and getComputedStyleMap

darrnshn opened the pull request:
(#350) Renames `Element.styleMap` to `Element.attributeStyleMap` and `Window.getComputedStyleMap` to `Element.computedStyleMap`. @shans  PTAL

shans commented:
Have a look at the resolution text in the linked issue. What we ended up deciding on is quite different to this.

darrnshn commented:
Cool, I've updated the patch. I removed the pseudo element parameter in `computedStyleMap` since it looks like it'll be handled by the pseudo element API or something? Also, there's a `CSSStyleRule.styleMap` which I didn't rename, should I?
